docs/running-a-node/ledger-management.md

```diff
@@ -12,7 +12,7 @@ The node automatically manages the full Nano ledger in the `data.ldb` file which
 
 --8<-- "directory-locations.md"
 
-This file will grow in size as the ledger does. As of April 2020 there are over 49 million blocks in the ledger which requires at least 26GB of free space. See [hardware recommendations](/running-a-node/node-setup/#hardware-recommendations) for more preferred node specs.
+This file will grow in size as the ledger does. As of September 2020 there are over 56 million blocks in the ledger which requires at least 37GB of free space. See [hardware recommendations](/running-a-node/node-setup/#hardware-recommendations) for more preferred node specs.
```

Comment:
@matm Thanks for this update. We are seeing ledgers taking up as little as 29GB of space under normal operations (see https://nanocrawler.cc/status for example), so that looks like a more appropriate figure here. If you can update we will get this merged in!

Reply:
Yes, done!